make/modules/java.base/Java.gmk

```diff
@@ -34,6 +36,8 @@ EXCLUDE_FILES += \
   $(TOPDIR)/src/java.base/share/classes/jdk/internal/module/ModuleLoaderMap.java
 
 EXCLUDES += java/lang/doc-files
+EXCLUDES += jdk/classfile/snippet-files
```

Comment:
... and the same goes for the `EXCLUDES`. If the line gets longer than 80 chars, split it like the `COPY` line above.

Reply:
fixed, thanks